blueprint/index.md

```diff
@@ -1 +1,210 @@
+
+## Contents
+* [Solution components](#solution-components)
+* [Prerequisites](#prerequisites)
+* [Implementation steps](#implementation-steps)
+* [Additional resources](#additional-resources)
+
+## Solution components
+
+* **Genesys Cloud** - A suite of Genesys Cloud services for enterprise-grade communications, collaboration, and contact center management. In this solution, you use an Architect inbound email flow, integration, data action, queues, and email configuration in Genesys Cloud.
+* **Amazon API Gateway** - An AWS service for using APIs in a secure and scalable environment. In this solution, the API Gateway exposes a REST endpoint that is protected by an API key. Requests that come to the gateway are forwarded to an AWS Lambda.
+* **AWS Lambda** - A serverless computing service for running code without creating or maintaining the underlying infrastructure. In this solution, AWS Lambda processes requests that come through the Amazon API Gateway and calls the Amazon Comprehend endpoint.
+* **AWS DynamoDB** - A fully managed NoSQL database service offered by Amazon Web Services. The service supports key-valued cloud services and data structures designed to run high-performance applications at any scale.
+  
+## Prerequisites
+
+### Specialized knowledge
+* Administrator-level knowledge of Genesys Cloud.
+* AWS Cloud Practitioner-level knowledge of AWS IAM, Amazon API Gateway, AWS Lambda, and Amazon DynamoDB.
+  
+### Genesys Cloud account
+
+* A Genesys Cloud license. For more information, see [Genesys Cloud Pricing](https://www.genesys.com/pricing "Opens the Genesys Cloud pricing page") in the Genesys website.
+* Master Admin role. For more information, see [Roles and permissions overview](https://help.mypurecloud.com/?p=24360 "Opens the Roles and permissions overview article") in the Genesys Cloud Resource Center.
+
+### AWS account
+* An administrator account with permissions to access the following services:
+  * AWS Identity and Access Management (IAM)
+  * Amazon API Gateway
+  * Amazon DynamoDB
+  * AWS Lambda
+* AWS credentials. For more information about setting up your AWS credentials on your local machine, see [About credential providers](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html "Opens the About credential providers page") in AWS documentation.
+* AWS CLI. For more information about installing the AWS CLI on your local machine, see [About credential providers](https://aws.amazon.com/cli/ "Opens the About credential providers page") in the AWS documentation.
+
+## Implementation steps
+
+1. Clone the GitHub repository
+
+### Clone the GitHub repository
+Clone the GitHub repository [ionic-mobile-open-messaging-blueprint](https://github.com/GenesysCloudBlueprints/ionic-mobile-open-messaging-blueprint "Opens the ionic-mobile-open-messaging-blueprint repository). The repository folder includes the sample configuration settings and folder structure for the front-end and back-end of the application. 
```

Comment:
You are missing the closing quote on the link

Reply:
fixed